Localization/de-DE/home.yaml

```
@@ -9,7 +9,7 @@ text: [
 
 # Associated text
 #"Regardless if you are already folding or haven't heard a word about it before, we need your help to reach our goal -- which is 1 million folders.",
-"Egal, ob Sie bereits falten oder noch kein Wort davon gehört haben: Wir brauchen Ihre Hilfe, um unser Ziel – eine Million Faltende – zu erreichen",
+"Egal, ob Sie bereits mitmachen oder noch nie davon gehört haben: Wir brauchen IHRE Hilfe, um unser Ziel – eine Million Faltende – zu erreichen",
```

```suggestion
"Egal, ob Sie bereits mitfalten oder noch nie davon gehört haben: Wir brauchen IHRE Hilfe, um unser Ziel – eine Million Faltende – zu erreichen",
```
imo, falten has to be mentioned in the first part of the sentence. otherwise the "unser ziel - eine million faltende" comes a little out of nowhere.